README.MD

```diff
@@ -13,9 +13,14 @@ This is the Go version of the [onboarding exercise](https://github.com/wexinc/ps
 - [Viper](https://github.com/spf13/viper)
 - [Logrus](https://github.com/Sirupsen/logrus)
 
+## Technical Proposal
+-[Swagger](https://docs.google.com/document/d/1K1WaCZhHi35BbvQ7UAGWOlJeGBoIlLVXpW7EQ8ryFdw/edit?usp=share_link)
+
 ## Pre-requisites
-- Docker
-- Go
+- [Go](https://golang.org/doc/install)
+- [Docker](https://docs.docker.com/manuals/)
+- [Docker-compose](https://docs.docker.com/compose/)
+- [Swagger](https://github.com/swaggo/gin-swagger)
 
 ## Getting started
 To run the application, make sure you have Docker and Go installed.
```

Comment:
This needs to be updated to insure you have swagger installed as well. Since it's referenced along with other libraries it's not clear that you have to install a command-line tool.

Reply:
The README file have a pre-requisite of things that you must have to run it, It have go, Docker, Docker-compose and Swagger. Each link will take you to the installation guide of each tool.

Reply:
In the instructions to run it it says "To run the application, make sure you have Docker and Go installed." THis statement does not reference Swagger at all.

Reply:
Updated